
# Frage 1 – Notebook 04: Tests & Auswertung

Dieses Notebook führt die **statistischen Tests** durch und druckt die Kernaussagen:
- **Binomialtest**: Ob die Mehrheit der Befragten einen *Anstieg* des Warenkorbwerts wahrnimmt.
- Zusätzliche Kennzahlen: 5‑Jahres‑Wachstum nominal/real.


In [3]:

import pandas as pd
import numpy as np
from scipy.stats import binomtest
from pathlib import Path

root = Path.cwd()
df5 = pd.read_csv(root / "warenkorb_auswertung_5J.csv")
ver = pd.read_csv(root / "umfrage_warenkorb_verteilung.csv")

def growth_pct(series):
    s0, sN = float(series.iloc[0]), float(series.iloc[-1])
    return (sN / s0 - 1) * 100

# 5‑Jahres‑Wachstum (real) LM
g_real = growth_pct(df5["LM_Umsatz_real"])
g_nom  = growth_pct(df5["LM_Umsatz_nom"])

# Binomialtest: "gestiegen" vs. Rest
k = int(ver.loc[ver["Antwort"].isin(["Deutlich gestiegen","Etwas gestiegen"]), "Anzahl"].sum())
n = int(ver["Anzahl"].sum())
p0 = 0.5

res = binomtest(k=k, n=n, p=p0, alternative="greater")

print("--- Ergebnisse ---")
print(f"5‑J‑Wachstum LM nominal: {g_nom:.1f}% | real: {g_real:.1f}%")
print(f"Umfrage: {k}/{n} = {k/n*100:.1f}% sehen einen Anstieg.")
print(f"Binomialtest H0: p ≤ {p0:.2f}  -> p‑Wert = {res.pvalue:.4f}")
if res.pvalue < 0.05:
    print("→ Signifikant: Mehr als die Hälfte nimmt einen Anstieg wahr.")
else:
    print("→ Nicht signifikant auf 5%-Niveau.")


--- Ergebnisse ---
5‑J‑Wachstum LM nominal: 23.3% | real: 17.1%
Umfrage: 26/39 = 66.7% sehen einen Anstieg.
Binomialtest H0: p ≤ 0.50  -> p‑Wert = 0.0266
→ Signifikant: Mehr als die Hälfte nimmt einen Anstieg wahr.


# Hypothese
Hypothese H0 (Nullhypothese):
Die Mehrheit der Online-Käufer nimmt keine Erhöhung des durchschnittlichen Warenkorbwertes in den letzten fünf Jahren wahr (p ≤ 0,5).

Hypothese H1 (Alternativhypothese):
Die Mehrheit der Online-Käufer nimmt eine Erhöhung des durchschnittlichen Warenkorbwertes in den letzten fünf Jahren wahr (p > 0,5).

Test: Binomialtest (einseitig, n = 39, k = 26).

Ergebnis: p = 0.0266 → Signifikant → H0 verworfen → H1 bestätigt.

Interpretation: Mehr als die Hälfte der Befragten sieht einen steigenden Warenkorbwert; die subjektive Wahrnehmung deckt sich mit den realen Statista-Daten (+17 % real).